config/cht/targets.js

```diff
@@ -0,0 +1,187 @@
+const extras = require('./nools-extras');
+const FIVE_YEARS_IN_DAYS = 1826;
+const {
+  today,
+  addDays,
+  isAlive,
+  getSubsequentPregnancyFollowUps,
+  isOnSameMonth,
+  getMostRecentLMPDateForPregnancy,
+  isActivePregnancy,
+  countANCFacilityVisits
+} = extras;
+
+module.exports = [
+
+  {
+    id: 'newborn-mortality',
+    type: 'percent',
+    icon: 'icon-death-neonatal',
+    goal: 0,
+    translation_key: 'targets.anc.newborn_mortality.title',
+    subtitle_translation_key: 'targets.all_time.subtitle',
+    appliesTo: 'contacts',
+    appliesToType: ['person'],
+    appliesIf: function (c) {
+      //console.log('a', c.contact.name, c.contact.date_of_birth, addDays(c.contact.date_of_birth, FIVE_YEARS_IN_DAYS).getTime(), today, addDays(c.contact.date_of_birth, FIVE_YEARS_IN_DAYS) > today);
+      return c && c.contact.date_of_birth && addDays(c.contact.date_of_birth, FIVE_YEARS_IN_DAYS) > today;
+    },
+    passesIf: function (c) {
+      //console.log('p', c.contact.name, c.contact.date_of_death, addDays(c.contact.date_of_birth, FIVE_YEARS_IN_DAYS) > new Date(c.contact.date_of_death));
+      return c.contact.date_of_death && addDays(c.contact.date_of_birth, FIVE_YEARS_IN_DAYS) > new Date(c.contact.date_of_death);
```

Comment:
Aren't all goals a fixed value which you achieve by getting more than the goal? Maybe I'm reading this wrong, but does this goal pass when there is >0 deaths of children under 5? It seems semantically backwards.

Reply:
The requirement spec says:
"I want to show under-5 mortality within CHW areas as a percentage of all children under 5 (in their area). Under-5 mortality means deceased children that were under the age of 5 at the time of their death. This is an all-time measure. The goal should be 0%."
If there were 20 children under-5 in the area and 2 have been deceased, the target should show as 10%. The goal is to keep it at 0%, so it is a backward goal.